spec/milestone_checker_spec.rb

```diff
@@ -186,7 +186,7 @@ module Danger
         it "reports a warning when asked to do so when a PR doesn't have a milestone set" do
           allow(@plugin.github).to receive(:pr_json).and_return({})
 
-          @plugin.check_milestone_due_date(if_no_milestone: :warn)
+          @plugin.check_milestone_due_date(report_if_no_milestone: :warning)
```

Comment:
`report_if_no_milestone` sounds like a yes/no question to me.

With Ruby being a weakly typed language, my first instinct would be to pass `true` and I would learn it's incorrect only later on (_this of course assumes my text editor / IDE is not smart enough to check that for me and that I jumped into coding without checking the API docs or source..._)

What do you think of taking a page from the [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/#weak-type-information) recommendation to precede each weakly typed parameter, such as `Any`, `Int`, or `String, with a noun describing its role:

<img width="500" alt="screen shot 2019-02-06 at 1 43 58 pm" src="https://user-images.githubusercontent.com/1218433/52317755-78bf6600-2a15-11e9-93ba-ec4a22fb71cb.png">

```diff
-          @plugin.check_milestone_due_date(report_if_no_milestone: :warning)
+         @plugin.check_milestone_due_date(report_type_if_no_milestone: :warning)
```

Reply:
Good point! Fixed on 5db5ee68fa5a4649694535eee6947f5cb1d45fbd.